src/DSE.Open.Values/Identifier.cs

```diff
@@ -1,25 +1,20 @@
 // Copyright (c) Down Syndrome Education International and Contributors. All Rights Reserved.
 // Down Syndrome Education International and Contributors licence this file to you under the MIT license.
 
+using System.Buffers;
 using System.Diagnostics;
-using System.Diagnostics.CodeAnalysis;
 using System.Runtime.InteropServices;
 using System.Security.Cryptography;
+using System.Text;
 using System.Text.Json.Serialization;
 using DSE.Open.Values.Text.Json.Serialization;
 
 namespace DSE.Open.Values;
 
-/// <summary>
-/// A randomly-generated sequence of digits and letters, with an optional prefix,
-/// used to identify something.
-/// </summary>
+[ComparableValue]
+[JsonConverter(typeof(JsonUtf8SpanSerializableValueConverter<Identifier, AsciiString>))]
 [StructLayout(LayoutKind.Auto)]
-[JsonConverter(typeof(JsonStringIdentifierConverter))]
-public readonly record struct Identifier
-    : ISpanFormattable,
-      ISpanParsable<Identifier>,
-      IEquatable<Identifier>
+public readonly partial struct Identifier : IComparableValue<Identifier, AsciiString>, IUtf8SpanSerializable<Identifier>
```

Comment:
EquatableValue/IEquatableValue rather than comparable - although we can sort like a string, there is not really a sense in which an id is greater than or less than another. If we ever need sorting, we can implement an IdentifierComparer.

Reply:
Done.